mentat/code_map.py

```diff
@@ -6,10 +6,10 @@
 from pathlib import Path
 from typing import Any
 
+CTAG = tuple[str | None, str, str, str | None, int]
```

Comment:
I like this having its own class. Does it make sense to label these fields, just for clarity - maybe with attr or namedtuple or something:

Reply:
I think it makes more sense as a type; it's not like we're doing anything with it other than getting the internals